pandas/tslib.pyx

```diff
@@ -3849,6 +3849,7 @@ def get_time_micros(ndarray[int64_t] dtindex):
 
 
 @cython.wraparound(False)
+@cython.boundscheck(False)
 def get_date_field(ndarray[int64_t] dtindex, object field):
```

Comment:
hmm, tried that out, but cython doesn't seem to take a view of strings like that?  http://stackoverflow.com/questions/28203670/how-to-use-cython-typed-memoryviews-to-accept-strings-from-python